playbooks/common/openshift-cluster/upgrades/etcd/backup.yml

```diff
@@ -0,0 +1,73 @@
+- name: Backup etcd
+  hosts: etcd_hosts_to_backup
+  vars:
+    embedded_etcd: "{{ hostvars[groups.oo_first_master.0].openshift.master.embedded_etcd }}"
```

Comment:
Keep an eye on this here: https://github.com/openshift/openshift-ansible/pull/2730

I have pending changes that I need someone to look at but I suspect the above fails if for any reason, the fact cache is missing, because we didn't recalc appropriate facts during upgrade.